w3af/plugins/audit/global_redirect.py

```diff
@@ -54,6 +55,17 @@ def audit(self, freq, orig_response):
 
         :param freq: A FuzzableRequest object
         """
+
+        netloc = freq._uri.netloc
```

Comment:
I would use `freq.get_uri()` instead of `freq._uri`, you never know what the getter might do